api/server/newUserController.js

```diff
@@ -0,0 +1,30 @@
+const User = require("./userModel");
```

Comment:
Two things:

1: I have built a userController in groups-page branch that already has more functionality than this controller although admittedly your queries are more up-to-date than mine. I suggest that I update my controller to match the current standards instead of pushing this controller to master. Otherwise, it would all be redundant.

2: I think you should put the models in a "models" directory and controllers in a "controllers" directory. This way we can prevent future merge issues with my branch and this one and also keep things on the backend a bit more organized

Reply:
I agree, I made a branch yesterday to modify the structure of the api. Then, when you requested a code review I saw that you had already made the changes so I didn't make them again. I just pulled the feature/groups page onto my working branch.